data-structure/README.md

```diff
@@ -281,7 +300,7 @@ An example of a `Work`, and its physical and digital `Manifestation`s:
 //       E.g. The Work is registered first and then the Manifestations are linked back to the Work.
 
 // In JSON-LD
-// Creation
```

Comment:
> where we used Creation earlier, we probably meant Work. What do you think?

OK. Then you should update all other references to `Creation` in `data-structure/README.md` like:

- https://github.com/COALAIP/specs/pull/6/files#diff-2c60b6c8624af5210d23e3cadf3e799fR76
- ...